### ISSUE TITLE
8x8 cross-point switch

### ISSUE THREAD

**Issue by micbeldyga** [elhep/8x8-crosspoint-switch#1]
Initial specification:

- 8x TTL compatible input via SMA. Selectable 50Ω termination with LED indicator
- 8x TTL compatible output via SMA
- cross-point switch configuration - every input can be routed to every output
- Total jitter shouldn't exceed 1ps
- DIOT/EEM form factor, max 6HP
- cross-point switching control via DIOT backplane or dedicated USB.

**Comment by gkasprow** [comment]
@micbeldyga There is missing SMA connector mezzanine design

**Comment by micbeldyga** [comment]
> @micbeldyga There is missing SMA connector mezzanine design

Yes, I’m aware. I wanted to align on a main board design first before I start with the mezzanine, which won’t take longer than few hours to design
